packages/abi-utils/lib/arbitrary.test.ts

```diff
@@ -0,0 +1,55 @@
+import { testProp } from "jest-fast-check";
```

Comment:
This seems to be the new way of doing things, esp. with Jest. I'm kind of keen to move in this direction; are you opposed? I figure, since this is a new package, it's reasonably safe enough to break convention to try it out here.